### PR TITLE
Banner completed

### PR DESCRIPTION
**Pull request by AbuNavsa**
Banner for STOPPED Assessment Completed

-  Assessment Stopped Header displayed in place of "Flagged" when an application has a "STOPPED"  flag
- The banner summary also now has "Stopped" in place of the workflow_status if the flag is present
- Tests have been written to check if the UI is showing accurate to Flagged/Assessment Stopped heading
- Banner_summary now has a default parameter of flag=None, and the banner only adjust to the stopped flag is this flag is not None

### CODE REVIEW

File: app/assess/templates/assessor_tasklist.html [hunk 1]
```diff
@@ -17,15 +17,15 @@
     </div>
 
     {% if flag and g.user.highest_role in ("ASSESSOR", "LEAD_ASSESSOR") %}
-        {{ assessment_flag(flag, flag_user_info) }}
+        {{ assessment_flag(flag) }}
```
Comment (tferns) [inline app/assess/templates/assessor_tasklist.html:20]: Don't we need the user info here?

File: app/assess/routes.py [hunk 2]
```diff
@@ -289,7 +290,7 @@ def application(application_id):
         state=state,
         application_id=application_id,
         flag=flag,
-        flag_user_info=accounts.get(flag.user_id) if flag else None,
+        # flag_user_info=accounts.get(flag.user_id) if flag else None,
```
Comment (tferns) [inline app/assess/routes.py:293]: Should this be commented out?

File: app/assess/templates/macros/banner_summary.html [hunk 3]
```diff
@@ -13,7 +13,15 @@ <h3 class="govuk-body-l fsd-banner-content fsd-banner-collapse-padding">Project
                         name: {{ project_name }}</h3>
                     <h3 class="govuk-body-l fsd-banner-content">Total funding requested:
                         &pound;{{ "{:,.2f}".format(funding_amount_requested | float) }}</h3>
-                    <h3 class="govuk-body-l fsd-banner-content fsd-banner-collapse-padding">{{ workflow_status | status_to_human }}</h3>
+                    <h3 class="govuk-body-l fsd-banner-content fsd-banner-collapse-padding">
+                        {% if flag == None  %}
+                            {{ workflow_status | status_to_human }}
+                        {% else %}
+                            {{ flag.flag_type.name | capitalize }}
+                        {% endif %}
```
Comment (tferns) [inline app/assess/templates/macros/banner_summary.html:21]: I'd probably flip this and go with:
```suggestion
                        {% if flag %}
                            {{ flag.flag_type.name | capitalize }}
                        {% else %}
                             {{ workflow_status | status_to_human }}
                        {% endif %}
```

Seems more intuitive to me, but to each their own (not a big deal)

File: app/assess/templates/macros/banner_summary.html [hunk 4]
```diff
@@ -1,4 +1,4 @@
-{% macro banner_summary(fund_name, project_reference, project_name, funding_amount_requested, workflow_status) %}
+{% macro banner_summary(fund_name, project_reference, project_name, funding_amount_requested, workflow_status, flag=None) %}
```
Comment (tferns) [inline app/assess/templates/macros/banner_summary.html:1]: Did you also add a change somewhere that calls `banner_summary` with a flag? I can't see it in this PR

**Review by tferns — APPROVED** [review]
Changes look good but think you're missing passing flag to `banner_summary`

**Review by tferns — CHANGES_REQUESTED** [review]
Actually sorry to revoke the approve but I think we need `user_info` re-added too

File: app/assess/templates/assessor_tasklist.html (hunk 1 above)
Comment (AbuNavsa) [reply app/assess/templates/assessor_tasklist.html:20]: Yup, removed temporarily to test, thought I put it back!

File: app/assess/routes.py (hunk 2 above)
Comment (AbuNavsa) [reply app/assess/routes.py:293]: Will be removed

File: app/assess/templates/macros/banner_summary.html (hunk 3 above)
Comment (AbuNavsa) [reply app/assess/templates/macros/banner_summary.html:21]: You're right!

**Comment by AbuNavsa** [comment]
Done in new branch

**Pull request closed without merging**